pertpy/preprocessing/_guide_rna.py

```diff
@@ -109,6 +112,93 @@ def assign_to_max_guide(
 
         return None
 
+    def assign_mixture_model(
+        self,
+        adata: AnnData,
+        model: str = "Poisson_Gauss_Mixture",
```

Comment:
```suggestion
        model: Literal["poisson_gauss_mixture"] = "poisson_gauss_mixture",
```

lower case is probably nicer no?